zk-token-sdk/src/transcript.rs

```diff
@@ -56,11 +56,11 @@ pub trait TranscriptProtocol {
     /// Append a domain separator for zero-balance proof.
     fn zero_balance_proof_domain_sep(&mut self);
 
-    /// Append a domain separator for validity proof.
-    fn validity_proof_domain_sep(&mut self);
+    /// Append a domain separator for grouped ciphertext validity proof.
+    fn grouped_ciphertext_validity_proof_domain_sep(&mut self);
```

Comment:
Would be nice to spell out `separator`, in a separate PR

Reply:
Yep, sounds good! Will do!